re/plugins/sink/mainflux.go

```diff
@@ -0,0 +1,111 @@
+package main
```

Comment:
Do these need to be in a separate dirs? Maybe it would be better to call a dir `kuiper` that would hold 2 files: `source.go` and `sink.go`